edison/static/js/policy.controller.js

```diff
@@ -5,6 +5,61 @@ import { PolicyElementsModifier } from './policyElementsModifier.js'
 var policy = new Policy();
 var countCondition = 0;
 
+var policiesExamples = [];
+var policyTest1 = new Policy('Test Policy One', 'living room', 'light On, shutters Off', 'humidity < 40, tempetrue < 25');
+var policyTest2 = new Policy('Test Policy Two', 'kichen', 'shutters Off', 'humidity > 80, tempetrue between 30 50');
+policiesExamples.push(policyTest1);
+policiesExamples.push(policyTest2);
+
+function setCardBody(policy) {
+    var cardBody = createInitElement('div', 'card-body');
+    var cardTitle = createInitElement('h5', 'card_title');
+    var cardText = createInitElement('h5');
+
+    cardBody.appendChild(cardTitle);
+    cardBody.appendChild(cardText);
+    cardText.innerHTML = `this policy is setting to the ${policy.room} and the policy do ${policy.command}
+    with these conditions: ${policy.condition}`;
```

Comment:
maybe you should write something like : This policy is associated with ${policy.room} and responsible for ${policy.command} with these conditions: ${policy.condition}
also, does the word "room" is written in policy.room? if not i think you should also add after ${policy.room} the word room (${policy.room} room)

Reply:
Ok